### DISCUSSION TITLE
Stupid question, but where can I download colortool.exe?

### DISCUSSION THREAD

**Discussion by gilgongo** [microsoft/terminal#14105] [Q&A] [answered]
On the website it says "_You can find the tool inside of this [Github repository](https://github.com/microsoft/console)_." But where?

**Comment by j4james** [comment, marked as answer]
See here:
https://github.com/microsoft/terminal/tree/main/src/tools/ColorTool

There is a link in the README to the [latest release build](https://github.com/microsoft/terminal/releases/tag/1904.29002), but that's from 2019, so if you want all the most recent changes you'd probably need to build it yourself.
